### PR TITLE
Add Licence

### PR DESCRIPTION
**Pull request by 1-alex98**
Took my name as I was not sure if u wanna be mentioned with your real name. 

Please confirm you wanna add MIT Liscene to this project.

### CODE REVIEW

**Comment by n3roxis** [comment]
I think using my real name is appropriate. You can do so.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020 Alexander von Trostorff
```
Comment (n3roxis) [inline LICENSE]: Copyright (c) 2020 Peter Ziegler

**Review by n3roxis — CHANGES_REQUESTED** [review]

**Comment by 1-alex98** [comment]
@Geosearchef @Rackover a small message that u are ok with that would be awesome :D

**Comment by Rackover** [comment]
We can have multiple names on the license, so I do not mind whether or not only Neroxis' name is credited or if we put our three/four names on the License. I think it is up to Neroxis to decide.

Also, you do not need my permission to do something for FAF 😛
Please stop pinging me, my mail box is already getting a lot of notifications from FAF github that I must disable every time independantly. 

If you need my name for credits purposes, use Roxanne De Louvegarde <rackover@racknet.noip.me>

**Comment by 1-alex98** [comment]
Sure u need to agree to a license

**Review by n3roxis — APPROVED** [review]

**Pull request merged**